docs/ref/request-response.txt

```diff
@@ -931,6 +931,9 @@ JsonResponse objects
    serialize the data. See :ref:`JSON serialization
    <serialization-formats-json>` for more details about this serializer.
 
+   The ``json_dumps_params`` is a dictionary used to pass named arguments to
```

Comment:
Need a `.. versionchanged:: 1.9` annotation (see elsewhere for an example) and release notes.